.github/workflows/test.yml

```diff
@@ -13,17 +13,18 @@ jobs:
 
   changelog-check:
     runs-on: ubuntu-20.04
-    if: ${{ github.event_name == 'pull_request' }}
 
     env:
-      PR_BODY: ${{ github.event.pull_request.body }}
+      PR_BODY: ${{ github.event.pull_re_quest.body }}
```

Comment:
this change looks like a typo?

Reply:
This was my test if the pipeline aborts on unknown structure references because on master the pull_request isn't available.